### PR TITLE
Added the Colour enhancement filter

### LINKED ISSUE #1804: Add a colour boost filter
**New Feature**

Add a new filter that enhances the overall colour of the image and kind of brightens the image a bit.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by batbrain7**
Fixed #1804 

Changes: Added the colour boost filter writing the C++ code for it.

Screenshots for the change: 
Before : 
screenshot_20180205-112726
After : 
screenshot_20180205-112717

### CODE REVIEW

**Comment by batbrain7** [comment]
@isuruAb @anantprsd5 please review and merge!!

**Comment by batbrain7** [comment]
@abishekvashok please review

File: app/src/main/jni/main_processing.cpp [hunk 1]
```diff
@@ -67,8 +67,13 @@ extern "C" {
...
                 applyBlueBoostEffect(src,dst,val);
                 break;
+            case 16:
+            applyColorBoostEffect(src,dst,val);
+                break;
+            
             default:
-                int lowThreshold = val;
+
+            int lowThreshold = val;
```
Comment (abishekvashok) [inline app/src/main/jni/main_processing.cpp]: Please don't change the intendation

File: app/src/main/jni/main_processing.cpp [hunk 2]
```diff
@@ -67,8 +67,13 @@ extern "C" {
             case 15:
                 applyBlueBoostEffect(src,dst,val);
                 break;
+            case 16:
+            applyColorBoostEffect(src,dst,val);
```
Comment (abishekvashok) [inline app/src/main/jni/main_processing.cpp]: Add a soft tab here to properly intend the lines.

**Comment by batbrain7** [comment]
@abishekvashok made the changes as suggested, please review

**Comment by batbrain7** [comment]
@abishekvashok made the changes please review

**Comment by abishekvashok** [comment]
Lets wait for Travis.

**Comment by batbrain7** [comment]
@abishekvashok @isuruAb please merge this approved PR!!

**Pull request merged**